src/server.h

```diff
@@ -725,8 +727,14 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 		Sounds
 	*/
 	std::unordered_map<s32, ServerPlayingSound> m_playing_sounds;
+	// Index for attached sound IDs by object ID. Must be kept in sync with m_playing_sounds.
+	std::unordered_multimap<u16, s32> m_attached_sound_ids;
 	s32 m_playing_sounds_id_last_used = 0; // positive values only
 	s32 nextSoundId();
+	std::pair<
```

Comment:
Not important... But would it be nicer to typedef this elaborate somewhere. It's unsightly long here :)

Reply:
I think I'll refactor this to pass the two iterators as pointers, which lets me update them (if the pointers are not null).